codec-http2/src/main/resources/META-INF/native-image/io.netty/codec-http2/native-image.properties

```diff
@@ -0,0 +1,16 @@
+# Copyright 2019 The Netty Project
+#
+# The Netty Project licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+Args =  --rerun-class-initialization-at-runtime=io.netty.handler.codec.http2.Http2CodecUtil \
```

Comment:
What makes these classes so special?

Reply:
@pmlopes

Reply:
I didn't went on the full root cause but it seems that one of the static initializers stores state that is specific to the current running process. This is the less intrusive fix, just rerun the static initializers again at runtime

Reply:
If I remember correctly the errors I think they are related to having direct byte buffers referenced from static variables. This will point to the wrong memory location at runtime as the heap is loaded with the references from the memory at compile time.